app/domains/fluggastrechte/components/StreitbeilegungCard.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Translations } from "~/services/translations/getTranslationByKey";
+import SummaryDataOverviewCard from "./SummaryDataOverviewCard";
+import type { FluggastrechtContext } from "../formular/context";
+import { FLOW_ID } from "../services/summaryPage/getOverviewData";
+
+const TITLE = "Außergerichtliche Streitbeilegung";
+
+function getReasonsAgainstStreitbeilegung(
+  userData: FluggastrechtContext,
+): string {
+  if (userData.streitbeilegungGruende === "no")
+    return "Der Versuch einer außergerichtlichen Streitbeilegung hat nicht stattgefunden und es bestehen keine Bedenken dagegen.";
+  if (userData.streitbeilegungGruende === "yesAirlineAgainst")
+    return "Der Versuch einer außergerichtlichen Streitbeilegung hat nicht stattgefunden und es bestehen Bedenken, weil das bisherige Verhalten der Fluggesellschaft dagegen spricht.";
+  if (userData.streitbeilegungGruende === "yesOtherReasons")
+    return "Der Versuch einer außergerichtlichen Streitbeilegung hat nicht stattgefunden und es bestehen Bedenken dagegen.";
+  return "";
+}
+function StreitbeilegungCard({
+  userData,
```

Comment:
```suggestion
  { streitbeilegung, streitbeilegungGruende, },
```
also we don't want to pass around `userData` I'd say we just take what we need